Remotion/Web/Development.WebTesting.IntegrationTests.Infrastructure/CustomWebTestConfigurationFactory.cs

```diff
@@ -20,52 +20,72 @@
 using System.IO.Compression;
 using Remotion.Web.Development.WebTesting.Configuration;
 using Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chrome;
+using Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chromium;
+using Remotion.Web.Development.WebTesting.WebDriver.Configuration.Edge;
 
 namespace Remotion.Web.Development.WebTesting.IntegrationTests.Infrastructure
 {
   public class CustomWebTestConfigurationFactory : WebTestConfigurationFactory
   {
-    protected override IChromeConfiguration CreateChromeConfiguration (WebTestConfigurationSection configSettings)
+    protected override IChromiumConfiguration CreateChromeConfiguration (WebTestConfigurationSection configSettings)
     {
       var chromeVersionArchivePath = ConfigurationManager.AppSettings["ChromeVersionArchive"];
-      var advancedChromeOptions = new AdvancedChromeOptions();
+      var advancedChromeOptions = new AdvancedChromiumOptions();
```

Comment:
When do we drop the AdvancedChromiumOptions?

Reply:
@MichaelKetting [RM-7265](https://re-motion.atlassian.net/browse/RM-7265) - I guess this should happen before Edge then?

Reply:
Yeah, keeps us from going in circles